source/installing.rst

```diff
@@ -0,0 +1,233 @@
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo apt-get install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Percona XtraBackup
+      $ sudo apt-get install percona-xtrabackup-80
+      $ #Install Percona Toolkit
+      $ sudo apt-get install percona-toolkit
+      $ #Install Orchestrator
+      $ sudo apt-get install percona-orchestrator
+      $ #Install MySQL Shell
+      $ sudo apt-get install percona-mysql-shell
+      $ #Install ProxySQL
+      $ sudo apt-get install proxysql2
+      $ #Install MySQL Router
+      $ sudo apt-get install percona-mysql-router  
+      
+.. seealso::
+
+   Percona Documentation:
+       * `Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation/apt_repo.html>`_
+       * `Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation/apt_repo.html>`_
+       * `Percona Toolkit <https://www.percona.com/doc/percona-toolkit/2.2/installation.html>`_
+ 
+
+Install |PS|-based deployment on |RHEL| / |centos| 
+--------------------------------------------------------------------------------
+
+.. admonition:: Platform Specific Notes
+
+  On |centos| 7, install the epel-release package. It includes the dependencies required to install Orchestrator. Use the following command:
+
+   .. code-block:: bash
+
+      $ sudo yum -y install epel-release
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo yum install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Orchestrator
+      $ sudo yum install percona-orchestrator
+      $ #Install Percona XtraBackup
+      $ sudo yum install percona-xtrabackup-80
+      $ #Install Percona Toolkit
+      $ sudo yum install percona-toolkit
+      $ #Install MySQL Shell
+      $ sudo yum install percona-mysql-shell
+      $ #Install ProxySQL
+      $ sudo yum install proxysql2
+      $ #Install MySQL Router
+      $ sudo yum install percona-mysql-router  
+
+.. seealso::
+
+   Percona Documentation:
+       * `Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation/yum_repo.html>`_
+       * `Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation/yum_repo.html>`_
+       * `Percona Toolkit <https://www.percona.com/doc/percona-toolkit/2.2/installation.html>`_
+
+Install |PXC|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PXC|:
+   
+   .. code-block:: bash
+   
+      $ sudo apt-get install percona-xtradb-cluster
+
+
+#. Install HAProxy:  
+
+   .. code-block:: bash
+   
+      $ sudo apt-get install percona-haproxy
```

Comment:
Full list of packages:
    - percona-haproxy
    - percona-haproxy-doc
    - percona-vim-haproxy
    - percona-haproxy-dbgsym

Reply:
Same question, is it enough to install percona-haproxy only to work with it? Or we must mention / install the full list of packages?